README.md

```diff
@@ -109,6 +109,30 @@ And similarly `zone_my_reverse_inaddr_arpa` and `zone_my_reverse_ip6_arpa` for I
 
 * deploy role to your servers
 
+### DDNS updates
+
+## Key generation
+
+If you want to have your DDNS keys created by this role configure `bind9_generate_ddns_key`:
+
+```yaml
+-  bind9_generate_ddns_key: true
+```
+
+Keys will by default be stored to `files/bind/zones` withinin your playbook location, but you can customize this with `bind9_local_keydir`
+
+```yaml
+-  bind9_local_keydir: true
```

Comment:
This seems to be a weird value for the variable :wink:

Reply:
Yup. Copy-paste FUBAR

Reply:
I modified the value into something more sensible